frame/contracts/src/debug.rs

```diff
@@ -0,0 +1,53 @@
+pub use crate::exec::ExportedFunction;
+use crate::{CodeHash, Config, LOG_TARGET};
+use pallet_contracts_primitives::ExecReturnValue;
+
+/// Umbrella trait for all interfaces that serves for debugging.
+pub trait Debugging<T: Config>: Tracing<T> {}
```

Comment:
Debug would be nicer but a bit annoying as it clash with `std:fmt:Debug`, `ConractDebug` would be another good name, any pref?

Reply:
`Debugger`?

Reply:
I like `Debugger`

Reply:
Will update and follow consensus